### DISCUSSION TITLE
How to add views count in single post

### DISCUSSION THREAD

**Discussion by booncu** [microweber/microweber#914] [General]
Hello i try to add views count in single post with module site_stats
when i embed this module not working in frontend and than show zero (0) value views, i dont no how to use this module for frontend.

<img width="360" alt="Screen Shot 2022-08-05 at 10 47 24" src="https://user-images.githubusercontent.com/10230514/182997644-ead1eeb9-dc0e-47a4-ae31-245dffbe3a54.png">

any have ideas for this, thanks for your time

**Comment by peter-mw** [comment]
Hello, 

you can add inside your template the module `site_stats/view_count`

`<module type="site_stats/view_count" content-id="your_content_id"  /> `

or with the php function

`print stats_get_views_count_for_content($content_id);`

**Reply by booncu** [reply]
wow it works

thanks for your time

i love bootstrap 5 and i reusable this template with my design, i hope microweber it's growth faster
